awx/ui/test/e2e/tests/test-xss.js

```diff
@@ -54,13 +59,15 @@ module.exports = {
 
                 urls.organization = `${pages.organizations.url()}/${data.organization.id}`;
                 urls.inventory = `${pages.inventories.url()}/inventory/${data.inventory.id}`;
+                urls.hosts = `${pages.hosts.url()}`;
                 urls.inventoryScript = `${pages.inventoryScripts.url()}/${data.inventoryScript.id}`;
                 urls.inventorySource = `${urls.inventory}/inventory_sources/edit/${data.inventorySource.id}`;
                 urls.sourceSchedule = `${urls.inventorySource}/schedules/${data.sourceSchedule.id}`;
                 urls.smartInventory = `${pages.inventories.url()}/smart/${data.smartInventory.id}`;
                 urls.project = `${pages.projects.url()}/${data.project.id}`;
                 urls.credential = `${pages.credentials.url()}/${data.credential.id}`;
                 urls.jobTemplate = `${pages.templates.url()}/job_template/${data.jobTemplate.id}`;
+                urls.jobTemplates = `${pages.templates.url()}`;
```

Comment:
I'm not seeing where this additional url (urls.jobTemplates) gets used in the test

Reply:
I think this was a stream crossing. The other xss, I thought needed the job templates tab since the schedules button/tab is usually gotten to by first visiting the job templates page. However, the UI is robust enough to go directly to that tab.

TL;DR will remove